htdocs/web_portal/controllers/downtime/downtime_utils.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+/*_____________________________________________________________________________
+ *=============================================================================
+ * File: downtime_utils.php
+ * Author: GOCDB DEV TEAM, STFC.
```

Comment:
we have #448 to remove authorship info from files, let's not add new info :)

Reply:
Ok. So, Only file and Description,  License information is needed right ?